tests/lib/publishedCohortLibraryGetAll.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Please use this format `my_file_name` and please rename it in order to contain expected_results. cheers 👍